File: ASSIGNMENT/C_/condition/q6.c
// q6 

#include<stdio.h>
int main(){
// Find the Character Is Vowel or Not
    char ch ;

    printf(" enter a character:");
    scanf("%c", &ch);


    if( ch == 'a' || ch =='A' || ch =='e' || ch == 'E' || ch == 'i' || ch == 'I' || ch == 'o' || ch == 'O' || ch == 'u' || ch == 'U'  ){
        printf("  character  is vowel  \n" , ch);

    } else{
        printf(" character   is consonant \n", ch);
    }
      return 0;
}

File: ASSIGNMENT/PYTHON_/Fundamentals of python/q6.py
#  Write a Python program to test whether a passed letter is a vowel or not. 
while(1):
    character=(input("enter character:")).lower()
    
    if character== 'a' or character=='e' or character=='i' or character=='o' or character== 'u':
        print("character is vowle--")
    else:
        print("consonants--")   
    choice=input("do you want check again(y/n):").lower()  
    if(choice!='y'):
       break     
print("thank you")